docs/user/hbase/visibilities.rst

```diff
@@ -46,8 +46,8 @@ When creating your data store, a default visibility can be configured for all fe
 .. code-block:: java
 
     Map<String, String> parameters = ...
-    parameters.put("security.enabled", "true");
-    parameters.put("visibilities", "admin&user");
+    parameters.put("hbase.security.enabled", "true");
+    parameters.put("geomesa.security.visibilities", "admin&user");
```

Comment:
Sanity check (I'm 90% sure about this):  The text representation for visibilities and auths work the *same* for Accumulo and HBase, right?

Reply:
heh, i'm not really sure... but that's what was in the docs to start with :P